ocr/single_character_detection.py

```diff
@@ -0,0 +1,125 @@
+import re
+
+from pytesseract import pytesseract
+
+from ocr.utils import resize_aspect_ratio
+import cv2
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+class SingleCharacterRecognition:
```

Comment:
Would be good to add docstrings to the methods in here as well.

Reply:
sure